metaflow/plugins/kubernetes/kubernetes_decorator.py

```diff
@@ -145,6 +145,12 @@ def step_init(self, flow, graph, step, decos, environment, flow_datastore, logge
                     "@kubernetes does not support parallel execution currently."
                 )
 
+            # If GPU count is specified, explicitly set it in self.attributes.
+            for k, v in deco.attributes.items():
+                if k == "gpu" and v != None:
```

Comment:
we can move this and the next line underneath L:166

Reply:
we can also do away with the code comment in L:165

Reply:
Done